core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ClassWarehouse.scala

```diff
@@ -187,19 +187,20 @@ class SQLExecutionInfoClass(
     var sqlCpuTimePercent: Double = -1)
 
 case class SQLAccumProfileResults(appIndex: Int, sqlID: Long, nodeID: Long,
-    nodeName: String, accumulatorId: Long, name: String, max_value: Long,
-    metricType: String, stageIds: String) extends ProfileResult {
+    nodeName: String, accumulatorId: Long, name: String, min: Long, median:Long,
+    max: Long, total: Long, metricType: String, stageIds: String) extends ProfileResult {
```

Comment:
Question:
Does this mean that before the changes, we used `max_value` to represent sum/total?

Reply:
Yes, it represented total earlier. i.e sum of all task metrics for a given Exec.